ROADMAP.md

```diff
@@ -0,0 +1,33 @@
+# Roadmap 2021
+
+![Roadmap Q1](./assets/roadmap_q1.png)
+
+In Q1 we focus entirely on using XayNet for the [Xayn app] in terms of federated learning and
+first simple analytics, such as gathering relevant AI performance data like [NDCG metrics]
+because we want to know how our AI models perform without violating the privacy of our users.
+As you know, our framework originated with the aim to aggregate machine learning models securely
+and privately between edge devices. Thereby, the models are transformed into one-dimensional lists
+so that at the end we only aggregate a list of numbers, so why not also aggregate other numerical
+analytics data, like AI performance metrics or user behaviour, such as screen times in our app,
+all of course with the privacy guarantees of XayNet. As such, we focus predominantly on mobile
+cross-device learning but also extend our framework to cover such use cases. In Q1 we take however
+mostly care about the internal mobile case and testing so we set the basis to further
+generalisation to external cases in the community during the rest of the year.
+
+![Roadmap Q2](./assets/roadmap_q2.png)
+
+In Q2 we have three main focus points: Extending XayNet to support also web applications, since
+also our [Xayn app] will be provided as a web version via [WASM]; integrating our product analytics
```

Comment:
😮 🤯 🚀

Reply:
😊 yeah quite excited about WASM
thanks to the nice abstraction of the sdk it shouldn't be that difficult (except for the libsodium stuff)